src/StackExchange.Redis/ConnectionMultiplexer.cs

```diff
@@ -2236,7 +2363,8 @@ public ConnectionMultiplexer GetSentinelMasterConnection(ConfigurationOptions co
                 config.EndPoints.Add(newMasterEndPoint);
             }
 
-            ConnectionMultiplexer connection = Connect(config, log);
+            ConnectionMultiplexer connection = ConnectImpl(config, log);
+            // TODO: Verify ROLE is master as specified here: https://redis.io/topics/sentinel-clients
```

Comment:
What's the status on this?

Reply:
It wasn’t doing this before and I did see that it’s kind of being verified by running an INFO call that gets the server mode.  But according to that link they recommend that you call ROLE after connect to verify the server is the right one. ROLE does not seem to be implemented and so I briefly looked at implementing it, but kind of just said never mind after I found out that it’s doing the INFO thing.

Reply:
Gotcha - separately filed #1451 to get that fixed up, thanks!